queries/erlang/highlights.scm

```diff
@@ -0,0 +1,75 @@
+[
+  "catch"
+  "of"
+  "case"
+  "when"
+  "after"
+  "receive"
+  "end"
+  "module"
+  "export"
```

Comment:
I don't erlang. maybe `module`, `export` fit well in `@include`. maybe some keywords are `@conditional`. I trust you in that regard